utils/profile.ts

```diff
@@ -104,26 +99,30 @@ export const isSenderAllowed = (
     readonly version: NonNegativeInteger;
   }
 ): TaskEither<CosmosErrors | IUnexpectedValue, boolean> =>
-  taskEither
-    .of<
+  pipe(
+    TE.of<
```

Comment:
We can safely remove typings

Reply:
aside of he previous comments we can write this with:
```
pipe(
    makeServicesPreferencesDocumentId(fiscalCode, serviceId, version),
    TE.of,
```

Reply:
Done [here](https://github.com/pagopa/io-functions-services/pull/154/commits/0dcbb5d5ddda89bbe680084525e265274f491207)